src/app/app.module.ts

```diff
@@ -3,17 +3,24 @@ import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { SelectLevelComponent } from './modules/level-select/level-select.component';
+import { LevelSelectModule } from './modules/level-select/level-select.module';
+import { LeftLayoutComponent } from './shared/components/left-layout/left-layout.component';
 
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    SelectLevelComponent,
+    LeftLayoutComponent
```

Comment:
It's bad practice to declare local components on an app.module level. Imagine, at least you have in your project about 20 components and you declare all of them in app module, it would look very massive. Thats why project on angular is divided into modules - to group relative between each other components and so on. And I already see module LevelSelectModule.

**BUT** it's look very wild to have one separate module just for this component. Which other relative components can be stored in this module? I can't imagine) If this component appears on a main page, you, for example, can create module for main page and store there all components, which will be on this page

Reply:
I have tried to declare LeftLayoutComponent in level-select component, but id did not work. So I don't know what's the problem. After I decided to declare in app module